### PR TITLE
update 2022 roadmap

### PR DESCRIPTION
**Pull request by mfanjie**
#### What type of PR is this?
Doc
#### What this PR does / why we need it:
Roadmap of 2022(H2)

### CODE REVIEW

File: docs/roadmaps/roadmap-2022.md [hunk 1]
```diff
@@ -20,15 +20,35 @@ Please let us know if you have urgent needs which are not presented in the plan.
...
 - Scalability to support 3k TSP and 3k EPA		
 - Algorithm and QoS Documentation
 - EHPA grafana dashboard
+- DSP Algorithm Optimization
 ### 0.7.0 [July]
-- Support apiservice router for multiple metric adapters
-- Prediction with Business Metrics
+- Support APIService router for multiple metric adapters
+- Prediction with business metrics
+- Recommendation Framework
```
Comment (qmhu) [inline docs/roadmaps/roadmap-2022.md:34]: - Support APIService router for multiple metric adapters
is replaced by `Support remote adapter for external metric` and released in 0.6.0
- Prediction with business metrics
is released in 0.6.0

File: docs/roadmaps/roadmap-2022.md [hunk 2]
```diff
@@ -20,15 +20,35 @@ Please let us know if you have urgent needs which are not presented in the plan.
...
+- Crane-Descheduler based on CPU/Memory metrics
+- Offline Algorithm Evaluator
 ### 0.8.0 [August]
-- Algorithm estimate notebook
+- Internal recommendation plugins
```
Comment (xieydd) [inline docs/roadmaps/roadmap-2022.md:38]: Internal -> External

**Review by xieydd — CHANGES_REQUESTED** [review]

**Review by qmhu — APPROVED** [review]

**Review by xieydd — APPROVED** [review]

**Pull request merged**